content/en/blog/_posts/2023-09-20-kubeadm-use-etcd-learner-mode.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: blog
+title: 'Kubeadm: Use ETCD Learner to join a control plane node safely'
+date: 2023-09-20
+slug: kubeadm-use-etcd-learner-mode
+---
+
+**Author:** Paco Xu(DaoCloud)
+
+Kubeadm now supports ETCD learner mode, which allows you to enhance the resilience and stability
+of your Kubernetes clusters by leveraging the [learner mode](https://etcd.io/docs/v3.4/learning/design-learner/#appendix-learner-implementation-in-v34) feature introduced in ETCD version 3.4.
+This guide will walk you through using ETCD learner mode with kubeadm.
+
+In v1.27, kubeadm introduced a new feature gate `EtcdLearnerMode`. With this feature gate enabled,
+when joining a new control plane node, a new etcd member will be created as a learner and 
+promoted to a voting member only after the etcd data are fully aligned.
+
+## What are the advantages of using ETCD learner mode?
```

Comment:
```suggestion
## What are the advantages of using etcd learner mode?
```
Do we need to use a consistent spelling with that in [etcd.io](https://etcd.io/).